src/index.css

```
@@ -538,6 +538,11 @@
     box-shadow: var(--droid-shadow, 0 10px 40px rgba(0, 0, 0, 0.35));
   }
 
+  .droid-tool-panel {
+    @apply overflow-hidden rounded-xl border border-droid-border bg-droid-surface;
+    box-shadow: var(--droid-shadow-sm, 0 4px 16px rgba(0, 0, 0, 0.28));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)